src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -615,3 +615,122 @@ app.use(function processItems(req, res, next) {
   })
 });
 ```
+
+### Vue.js
+
+Vue Tracing Integration allows us to track rendering performance during initial application load.
+
+It is achieved by injecting our handler inside Vue's `beforeCreate` mixin, which gives us an access to component during it is all lifecycle stages.
+When we encounter component named `root`, which is top-level Vue instance (as in `new Vue({})`), we use our APM Tracing integration,
```

Comment:
```suggestion
When we encounter component named `root`, which is a top-level Vue instance (as in `new Vue({})`), we use our AM Tracing integration,
```

Think we call it `AM ` instead of `APM` now.